runtime-modules/membership/src/lib.rs

```diff
@@ -216,16 +216,26 @@ decl_storage! {
     }
     add_extra_genesis {
         config(default_paid_membership_fee): BalanceOf<T>;
-        config(members) : Vec<(T::AccountId, String, String, String)>;
+        config(members) : Vec<genesis::Member<T::MemberId, T::AccountId, T::Moment>>;
         build(|config: &GenesisConfig<T>| {
-            for (who, handle, avatar_uri, about) in &config.members {
-                let user_info = ValidatedUserInfo {
-                    handle: handle.clone().into_bytes(),
-                    avatar_uri: avatar_uri.clone().into_bytes(),
-                    about: about.clone().into_bytes()
-                };
-
-                <Module<T>>::insert_member(&who, &user_info, EntryMethod::Genesis);
+            for member in &config.members {
+                let checked_user_info = <Module<T>>::check_user_registration_info(
+                    Some(member.handle.clone().into_bytes()),
+                    Some(member.avatar_uri.clone().into_bytes()),
+                    Some(member.about.clone().into_bytes())
+                ).expect("Importing Member Failed");
```

Comment:
We should avoid panicking in the runtime code. Last time we changed panicking to the printing error message.

Reply:
This code in the `build(|config: &GenesisConfig<T>| {  })` block isn't runtime code. It is only native code that runs when building the genesis chain-spec. So it is safe to panic.